ADOPTERS.md

```diff
@@ -0,0 +1,9 @@
+# Adopters
+
+_If you're using ORT in your organization, please add your organization's name to the table below. This really helps the project to gain momentum and credibility. It's a small contribution back to the project with a big impact._
```

Comment:
Would it make sense to say: please add your organization's name to the **bottom of** the table below? So that the ordering is defined?

Reply:
I think I would prefer alphabetical sorting, to not imply that the order has any meaning. Also I don't like that the list is not sorted at all in the linked example.

Reply:
Agree with @mnonnenmacher. So, @tsteenbe let's mention the ordering in the comment then?!

Reply:
@mnonnenmacher  @fviernau Changed the comment to say "to the alphabetically sorted table below."